service-sdk/12.0.0/migrate-to-open-api/messaging-api/pom.xml

```diff
@@ -91,6 +91,23 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
12.1.0 archetype doesn't generate this by default. Should we be asking users to add this plugin?